crates/biome_cli/tests/snapshots/main_cases_handle_vue_files/lint_vue_ts_files.snap

```diff
@@ -88,8 +88,12 @@ file.vue:4:8 lint/style/noInferrableTypes  FIXABLE  ━━━━━━━━━
   
   i Safe fix: Remove the type annotation.
   
-    4 │ var·foo:·string·=·"";
-      │        --------      
+    2 2 │   delete a.c;
+    3 3 │   
+    4   │ - var·foo:·string·=·"";
+      4 │ + var·foo·=·"";
+    5 5 │   
+  
```

Comment:
The diff seems less readable. Hev you some idea to solve this in a future PR?

Reply:
Yes I noticed this, but there're also cases (and seems more of them) where diffs become more "readable". For example: [this](https://github.com/biomejs/biome/pull/2237/files/f22ffa7904a2f2cbfc34fb370eca32197209a86a#diff-f82b7f387f85c41ab57a42ce3c76aa047f8eb35b533da0ad9ae8d229c7b76e80) and [this](https://github.com/biomejs/biome/pull/2237/files/f22ffa7904a2f2cbfc34fb370eca32197209a86a#diff-909870a56ad7bb50c13faddc9c18c869efce2897bf1c703a55f9e36d2fb22dc8).

I wasn't planning to solve it because 1) these kind of diffs already exist before this PR (like the improved ones above) 2) I actually don't find them to be less readable when they're colored in the terminal.

But I can take a look to it to find out why some diffs change their forms into one way or another. Though I'm not sure I will find a solution. 😂

Reply:
Thanks for the note! My eyes only noticed regressions 😅